docs/resources/security_monitoring_rule.md

```diff
@@ -134,6 +133,32 @@ Optional:
 - `learning_threshold` (Number) A number of occurrences after which signals are generated for values that weren't learned. Valid values are `0`, `1`. Defaults to `0`.
 
 
+<a id="nestedblock--options--third_party_rule_options"></a>
+### Nested Schema for `options.third_party_rule_options`
+
+Required:
+
+- `default_status` (String) Severity of the default rule case, when none of the third party cases match. Valid values are `info`, `low`, `medium`, `high`, `critical`.
```

Comment:
```suggestion
- `default_status` (String) Severity of the default rule case, when none of the third-party cases match. Valid values are `info`, `low`, `medium`, `high`, `critical`.
```

Reply:
Thanks! Applied in https://github.com/DataDog/terraform-provider-datadog/pull/2231/commits/1e4872ae66b9b61487da7cfbdc524101cda4f968.